source/reference/operator/query.txt

```diff
@@ -338,25 +352,17 @@ Projection Operators
 
```

Comment:
Can you bump the column width slightly so that the description field doesn't wrap by one character please?

Reply:
This is the problem we discussed before the break - sphynx is modifying the column widths. 

The settings here are the same as for the table above this one, however while that list-table fills the width of the page, this one only partially fills it. (See the 'Element' table above, it has the same problem. The same error appears on other index pages too.) 

None of these changes update the widths: 
-  Removing the " :class: singlecol " directive from line 11 
-  Changing  from 30, 70 to :widths: 25, 75
-  Adding a " :class: singlecol " directive to the list table
-  Various combinations of these